pkg/ruler/base/manager_metrics_test.go

```diff
@@ -14,12 +14,13 @@ import (
 
 	"github.com/grafana/loki/pkg/logproto"
 	"github.com/grafana/loki/pkg/ruler/rulespb"
+	"github.com/grafana/loki/pkg/util/constants"
 )
 
 func TestManagerMetricsWithRuleGroupLabel(t *testing.T) {
 	mainReg := prometheus.NewPedanticRegistry()
 
-	managerMetrics := NewManagerMetrics(false, nil)
+	managerMetrics := NewManagerMetrics(false, nil, constants.Cortex)
```

Comment:
intentional to leave it as cortex?

Reply:
Yes, this test checks the actual metrics output so I left this as Cortex to check it is passed through correctly. The following test uses Loki so that proves it's not hardcoded.